src/resolvers/records.js

```diff
@@ -100,8 +110,15 @@ module.exports = {
 			}
 
 		},
-		updateRecord: async (parent, { id }) => {
+		updateRecord: async (parent, { id },  { req }) => {
 
+			// In case of own site don't update
+			if (req.cookies && req.cookies.ackee_login === "1") {
```

Comment:
Let's create a small util for this as it's used in two places. `isLoggedIn(res)` => `true`/`false` in `src/utils/`.

Reply:
Good idea. Done.
One small change: Parameter name will be `req`, not `res`